aiida/orm/nodes/node.py

```diff
@@ -970,6 +968,11 @@ def store_all(self, with_transaction=True, use_cache=None):
 
         :parameter with_transaction: if False, do not use a transaction because the caller will already have opened one.
         """
+        if use_cache is not None:
```

Comment:
Since the ``use_cache`` argument is now just ignored, would it make sense to drop it before 1.0? Or is the interface for 1.0 frozen already?

Reply:
We froze the interface with `v1.0.0b3` and decided not to change it anymore. I doubt anyone was using it, but just to be consistent as much as possible with what we promised towards users

Reply:
One could argue that simply ignoring the argument is still a change in interface, since the semantics of the code changed.

Reply:
You are absolutely right, except in this case I think it is acceptable because the resulting behavior is identical to that before the change (with the exception that `Data` nodes stored with `use_cache=True` in the past would have gotten the `_aiida_cached_from` extra, which is no longer the case, but this extra is meaningless anyway in this scenario).

Reply:
Yeah, true -- since ``_aiida_cached_from`` is an internal extra, there's no public-facing change.